crates/fuel-core/src/database/block.rs

```diff
@@ -147,28 +151,36 @@ impl Database {
         )
         .next()
         .ok_or(DatabaseError::ChainUninitialized)?
-        .map(|(height, id): (Vec<u8>, Bytes32)| {
+        .map(|(height, id): (Vec<u8>, BlockId)| {
             let bytes = <[u8; 4]>::try_from(height.as_slice())
                 .expect("all block heights are stored with the correct amount of bytes");
             (u32::from_be_bytes(bytes).into(), id)
         })
     }
 
-    fn latest_block(&self) -> DatabaseResult<Option<(Vec<u8>, Bytes32)>> {
-        self.iter_all(
-            Column::FuelBlockIds,
-            None,
-            None,
-            Some(IterDirection::Reverse),
-        )
-        .next()
-        .transpose()
+    pub fn latest_block_ids(&self) -> DatabaseResult<Option<(BlockHeight, BlockId)>> {
+        let ids = self
+            .iter_all::<Vec<u8>, BlockId>(
+                Column::FuelBlockIds,
```

Comment:
realized the description of this column is somewhat misleading as it says:

> /// Maps fuel block id to fuel block hash

But the block id is the same thing as the hash. It should say:

> /// Maps fuel block height to fuel block id